Tools/CMake/AMReXSetDefines.cmake

```
@@ -81,11 +81,35 @@ if (AMReX_FORTRAN)
       $<$<COMPILE_LANGUAGE:Fortran>:BL_LANG_FORT AMREX_LANG_FORT>
       )
 
-    if (AMReX_MPI)
+   if (AMReX_MPI)
       target_compile_definitions( amrex PRIVATE
```

Would downstream codes depend on these compiler define flags or include `AMReX_Config.H` in their Fortran files?

If we want to make sure they also have the defines added, then we can make those `PUBLIC` definitions
```suggestion
      target_compile_definitions( amrex PUBLIC
```
(same below)